litmus-portal/graphql-server/pkg/usermanagement/create.go

```diff
@@ -45,6 +46,13 @@ func CreateUser(ctx context.Context, user model.CreateUserInput) (*model.User, e
 		return nil, err
 	}
 
+	publicHub := model.CreateMyHub{
+		HubName:    "Chaos Hub",
+		RepoURL:    "https://github.com/litmuschaos/chaos-charts",
+		RepoBranch: "master",
+	}
+	_, err = myhub.AddMyHub(ctx, publicHub, user.Username)
```

Comment:
Here here is not handle

Reply:
@rajdas98 the required change is done PTAL..